src/pages/search/audio.vue

```diff
@@ -69,16 +73,21 @@ export default defineComponent({
 
     const results = computed(() => props.resultItems.audio)
 
-    const isMinScreenMd = isMinScreen('md', { shouldPassInSSR: true })
+    const isMinScreenMd = inject(IsMinScreenMdKey)
+    const filterVisibleRef = toRef(props, 'isFilterVisible')
 
     // On SSR, we set the size to small if the User Agent is mobile, otherwise we set the size to medium.
     const isMobile = useBrowserIsMobile()
-    const audioTrackSize = computed(() => {
-      return !isMinScreenMd.value || isMobile
-        ? 's'
-        : props.isFilterVisible
-        ? 'l'
-        : 'm'
+    const audioTrackSize = ref(
```

Comment:
I think we'll need to revisit the audioTrackSize selection with @panchovm after the new_header is added. Currently, I feel like the `l`/`m` change on the `md` screen makes it look worse instead of better. And the code is fairly complicated.